src/main/java/sk/uniba/fmph/dcs/game_board/ArbitraryBuilding.java

```diff
@@ -0,0 +1,33 @@
+package sk.uniba.fmph.dcs.game_board;
+
+import sk.uniba.fmph.dcs.stone_age.Effect;
+
+import java.util.Collection;
+import java.util.OptionalInt;
+
+public final class ArbitraryBuilding implements Building {
+    private final int maxNumberOfResources;
+    private final static int MAX_NUMBER_ALLOWED = 7;
```

Comment:
```suggestion
    private static final int MAX_NUMBER_ALLOWED = 7;
```
Moja chyba, napísal som to naopak.